### ISSUE TITLE
Feature Request: Use existing ingame macro

### ISSUE THREAD

**Issue by fearkite** [PunishXIV/Artisan#108]
Hey ya'll! :D 

I was wondering if this was possible already? I might be derping it and can't find the option. The reason is that I'm using Craftimizer to 'copy' a macro to Macro Mate which in turn just makes a macro (or two). It's a one click process. Super convenient. But I'd like to use Artisans automatic crafting option with this, but it's very tedious copy and pasting macros (while I'm leveling). 

So, simply put: if Artisan could use an in game macro as an option.. that would be awesome!

**Comment by Taurenkey** [comment]
No, it's not something you can do automatically. I don't support Craftimizer for a number of reasons (it crashed with Artisan, results it generates were often illogical etc.) so I don't really see a need for a feature like this when Artisan has its own solver for levelling.
